api/processors/group.py

```diff
@@ -52,6 +59,33 @@ def _create_log_events_start(cls, group: Group) -> None:
             for teacher in group.teachers.iterator()
         )
 
+    @classmethod
+    def _create_log_events_abort(cls, group: Group) -> None:
+        GroupLogEvent.objects.create(group=group, type=GroupLogEventType.ABORTED)
+
+        CoordinatorLogEvent.objects.bulk_create(
+            CoordinatorLogEvent(
+                coordinator=coordinator,
+                group=group,
+                type=CoordinatorLogEventType.TOOK_NEW_GROUP,
+            )
+            for coordinator in group.coordinators.iterator()
+        )
+
+        StudentLogEvent.objects.bulk_create(
+            StudentLogEvent(
+                student=student, to_group=group, type=StudentLogEventType.GROUP_ABORTED
```

Comment:
```suggestion
                student=student, from_group=group, type=StudentLogEventType.GROUP_ABORTED
```
same for teachers. `to_group` is when somebody is added to a group. Since here group is aborted, they all exit the group, hence `from_group`.

(for transfer, both `from_group` and `to_group` will be filled)